speech/cloud-client/beta_snippets.py

```diff
@@ -156,21 +153,18 @@ def transcribe_file_with_diarization(speech_file):
 
     config = speech.types.RecognitionConfig(
         encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
-        sample_rate_hertz=16000,
         language_code='en-US',
         enable_speaker_diarization=True,
         diarization_speaker_count=2)
 
     print('Waiting for operation to complete...')
     response = client.recognize(config, audio)
 
-    for i, result in enumerate(response.results):
-        alternative = result.alternatives[0]
-        print('-' * 20)
-        print('First alternative of result {}: {}'
-              .format(i, alternative.transcript))
-        print('Speaker Tag for the first word: {}'
-              .format(alternative.words[0].speaker_tag))
+    result = response.results[-1]
+    words_info = result.alternatives[0].words
+    pieces = ['%s (%s)' % (word_info.word, word_info.speaker_tag)
+              for word_info in words_info]
+    print(' '.join(pieces))
```

Comment:
I feel this might not illustrate the "typical" use case, where the developer might more likely want to group and join the words according to their speaker_tag.

Reply:
Interesting point. Tough to say what the right use case is. But I see it just as a sample. to show them the API, and not the use case. Do you think we can keep it as is, or should we change it?

Reply:
I agree - in that sense perhaps let's just iterate through words_info and print everything without the nice formatting of `'%s (%s)'`.  the expected output of the test really confused me.

Reply:
I made some changes to make the output to look like this:

Speaker #1: I'm here
Speaker #2: hi I'd like to buy a Chrome Cast and I was wondering whether you could help me